tests/system/migrations/test_0042_update_workflow_timestamp.py

```diff
@@ -0,0 +1,126 @@
+def test_migration(write, finalize, assert_model):
+    write(
+        {
+            "type": "create",
+            "fqid": "motion/1",
+            "fields": {
+                "a": 1,
+                "created": 1686740704,
+            },
+        },
+        {
+            "type": "create",
+            "fqid": "motion_workflow/1",
+            "fields": {"b": 1, "set_created_timestamp": True},
+        },
+    )
+    write(
+        {
+            "type": "update",
+            "fqid": "motion/1",
+            "fields": {
+                "a": 2,
+                "created": 1686740814,
+            },
+        },
+        {
+            "type": "update",
+            "fqid": "motion_workflow/1",
```

Comment:
The flag is in motion_state, not motion_workflow.
I'm wondering that the test don't fail

Reply:
Yes, of course.